applications/nrf5340_audio/src/modules/hw_codec.c

```diff
@@ -316,3 +319,96 @@ int hw_codec_init(void)
 
 	return 0;
 }
+
+static int cmd_input(const struct shell *shell, size_t argc, char **argv)
+{
+	int ret;
+	uint8_t idx;
+
+	enum hw_codec_input {
+		LINE_IN,
+		PDM_MIC,
+		NUM_INPUT,
+	};
+
+	if (argc != 2) {
+		shell_error(shell, "Wrong number of arguments provided %d", argc);
```

Comment:
Let's be specific and say "Two arguments required"

Reply:
Hmm, isnt it actually one argument that is required from the user? And one other non-user input argument?

Reply:
Yes, only one argument required